src/components/App.js

```diff
@@ -1,8 +1,74 @@
+import React, { useState } from 'react';
+import SelectNumber from './Select';
+import Board from './Board';
+import { alphabet } from '../letters/Alphabet';
 
 function App() {
+  const [showBoard, setShowBoard] = useState(false);
+  const [maxNumberConsonants, setMaxNumberConsonants] = useState('');
+  const [wordLengths, setWordLengths] = useState([]);
+  const [remainingAlphabet, setRemainingAlphabet] = useState(alphabet);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [msgColorRed, setMsgColorRed] = useState(true);
+  const totalNumberOfConsonants = 27;
```

Comment:
This can be moved to a constants file if you'd like. Or at least moved to a constant at the top of the file.
The problem with defining it in `function App` is that means its re-created every render

Reply:
Created constants file.